rbac/management/role/view.py

```diff
@@ -349,3 +363,17 @@ def validate_and_get_access_list(self, data):
         for access in access_list:
             AccessSerializer(data=access).is_valid(raise_exception=True)
         return access_list
+
+    def validate_and_get_additional_field_key(self, params):
+        """Validate the add field key."""
+        field = params.get(ADDITIONAL_FIELD_KEY)
+        if field is not None and field not in VALID_FIELD_VALUES:
+            key = 'detail'
+            message = '{} query parameter value {} is invalid. booleans are valid inputs.'.format(
+                ADDITIONAL_FIELD_KEY,
+                field)
+            raise serializers.ValidationError({key: _(message)})
+        if field is None:
```

Comment:
This works, but we may also be able to use list comprehension here to remove the `None` value and avoid this conditional? Something like:

```python
all_fields = LIST_ROLE_FIELDS + [field]
return [f for f in all_fields if f]
```